src/diffusers/models/attention.py

```diff
@@ -1,10 +1,29 @@
 import math
-from typing import Optional
+import os
+from inspect import isfunction
+from typing import Any, Optional
 
 import torch
 import torch.nn.functional as F
 from torch import nn
 
+import xformers
+import xformers.ops
+from einops import rearrange
```

Comment:
We're trying to keep `diffusers` as light-weights as possible for now - could we maybe remove the `einops` dependency?